code/game/objects/items/fireaxe.dm

```diff
@@ -13,11 +13,12 @@
 	slot_flags = ITEM_SLOT_BACK
 	attack_verb = list("attacked", "chopped", "cleaved", "tore", "lacerated", "cut")
```

Comment:
since you're refactoring a bunch of stuff, we could probably make a bunch of these standardized based on the sharpness of the weapon. could see to that in a future PR if you don't feel like doing it